### ISSUE TITLE
Table prefixes are not picked up by autogenerate

### ISSUE THREAD

**Issue by CaselIT** [sqlalchemy/alembic#721]
**Describe the bug**
Currently sqlalchemy [`Table.prefixes`](https://docs.sqlalchemy.org/en/13/core/metadata.html#sqlalchemy.schema.Table.params.prefixes) are not picked up by alembic when generating a migration using 
`alembic revision --autogenerate`
**Expected behavior**
The `prefixes` are picked up by alembic. I've not tested it with other table keyboard, but maybe it's a more common theme.

**To Reproduce**
Please try to provide a [Minimal, Complete, and Verifiable](http://stackoverflow.com/help/mcve) example, with the migration script and/or the SQLAlchemy tables or models involved.
See also [Reporting Bugs](https://www.sqlalchemy.org/participate.html#bugs) on the website.

```py
# Add a new table like this one to the metadata
import sqlalchemy as sa

sa.Table(
    "foo",
    metadata,
    sa.Column("id", sa.Integer(), primary_key=True),
    sa.Column("other", sa.Integer),
    prefixes=["UNLOGGED"],
)

```

**Error**

```
# The reflected migration code generated is
def upgrade():
    # ### commands auto generated by Alembic - please adjust! ###
    op.create_table(
        "foo",
        sa.Column("id", sa.Integer(), nullable=False),
        sa.Column("other", sa.Integer(), nullable=True),
        sa.PrimaryKeyConstraint("id"),
    )
    # ### end Alembic commands ###
```

**Versions.**
 - OS: windows
 - Python: 3.7.7
 - Alembic: 1.4.2
 - SQLAlchemy: 1.3.18
 - Database: postgres
 - DBAPI: pycopg2

**Additional context**
Manually adding `prefixes=[...]` to the `create_table` function after autogenerate does work correctly, so this is just a nice to have feature

**Have a nice day!**

**Comment by rmoretto** [comment]
If no one is working in this issue I would like to contribute!

**Comment by CaselIT** [comment]
I don't think anyone is working on it. If you could contribute a pr it would be great!

**Comment by sqla-tester** [comment]
**rmoretto** has proposed a fix for this issue in the **master** branch:

**Added detection of Table prefixes in the create table render** https://gerrit.sqlalchemy.org/c/sqlalchemy/alembic/+/2212

**Comment by markfickett** [comment]
For posterity, fixed in alembic==1.5.0 [per release notes](https://github.com/sqlalchemy/alembic/releases/tag/rel_1_5_0).
